### DISCUSSION TITLE
Update procedure

### DISCUSSION THREAD

**Discussion by well-souza** [nilsteampassnet/TeamPass#4263] [General]
Team....
I am currently using version 3.0.0.17, but when carrying out the update procedure to the latest version, several errors occur, the main one being the install/upgrade link that remains stuck on the screen.
Is there any procedure for how to get to the version?
I performed the update test and the best I could get was version 3.0.0.23, the most recent ones, all showed the same error.

**Comment by alexyucra** [comment]
check the logs, I already had the same problems, check the configuration files too, the process does not progress if there are different domains.
